CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.3.0 (2016-2-19)
```

Comment:
Probably something went wrong during my rebase attempt. There aren't be any changes at all to this file from my side. I'll try to roll the changes to this file back (not sure how yet).